src/reportWebVitals.js

```diff
@@ -1,12 +1,12 @@
-const reportWebVitals = onPerfEntry => {
+const reportWebVitals = async (onPerfEntry) => {
   if (onPerfEntry && onPerfEntry instanceof Function) {
-    import('web-vitals').then(({ getCLS, getFID, getFCP, getLCP, getTTFB }) => {
-      getCLS(onPerfEntry);
-      getFID(onPerfEntry);
-      getFCP(onPerfEntry);
-      getLCP(onPerfEntry);
-      getTTFB(onPerfEntry);
-    });
+    const { getCLS, getFID, getFCP, getLCP, getTTFB } = await import('web-vitals');
+    
+    getCLS(onPerfEntry);
+    getFID(onPerfEntry);
+    getFCP(onPerfEntry);
+    getLCP(onPerfEntry);
```

Comment:
Any way to name these something different? Can't see the difference between FCP and LCP.